src/client/setup.ts

```diff
@@ -39,7 +43,16 @@ function twitter(): void {
     }
 }
 
+function performanceMetrics(): void {
+    window.addEventListener(
+        'load',
+        () => console.log('Metrics: ', metrics(performance.getEntries())),
```

Comment:
can we use the `logger` module Alex made? it's currently also using `console.log()` but it would be easy for us to swap that to a sentry log or something else

Reply:
Interesting question. Currently we plan to pass this information to the native layers, for them to bundle with the other metric information before sending off.

Do we plan to introduce any client-side logging frameworks?